src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue12126.cs

```diff
@@ -12,13 +12,17 @@ public Issue12126(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "[iOS] TabBarIsVisible = True/False breaking for multiple nested pages";
 
-	// Where does TapBackArrow() come from?
-	// [Test]
-	// [Category(UITestCategories.Shell)]
-	// public void NavigatingBackFromMultiplePushPagesChangesTabVisibilityCorrectly()
-	// {
-	// 	App.WaitForElement("TestReady");
-	// 	TapBackArrow();
-	// 	App.WaitForElement("Tab 1");
-	// }
+	
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void NavigatingBackFromMultiplePushPagesChangesTabVisibilityCorrectly()
+	{
+		App.WaitForElement("TestReady");
+		App.TapBackArrow();
+#if WINDOWS
+		App.WaitForElement("navViewItem");
```

Comment:
@jsuarezruiz is there a way we can make this work so it doesn't have to be an if/def? Do we need some additional extensions for tabs? Do we need to add some automationids at the platform level?

Reply:
I am going to take a look.

Reply:
cc @anandhan-rajagopal

Reply:
@anandhan-rajagopal Could you share more details on why you something different in Windows?

Reply:
In Windows, I'm encountering a timeout exception when using App.WaitForElement("Tab 1"), but using the x:Name for that element has worked successfully.

![image](https://github.com/user-attachments/assets/eb570e1f-a5d0-4775-a458-5824fc8d2df6)

Reply:
Ok, applied changes to fix the problem and not require compilation directives in the test.
<img width="372" alt="image" src="https://github.com/user-attachments/assets/5bb9bdd3-8e29-4c95-afa8-302d036fadf5">